cmd/context/create.go

```diff
@@ -237,7 +245,45 @@ func (_ *ContextUse) initKubernetesContext(ctxOptions *ContextOptions) error {
 	return nil
 }
 
-func (*ContextUse) initEnvVars() {
+func (c ContextCommand) getUserContext(ctx context.Context) (*types.UserContext, error) {
+	client, err := c.OktetoClientProvider.Provide()
+	if err != nil {
+		return nil, err
+	}
+
+	retries := 0
+	for retries <= 3 {
+		userContext, err := client.User().GetContext(ctx)
+
+		if err != nil && errors.IsForbidden(err) {
+			okteto.Context().Token = ""
+			if err := c.OktetoContextWriter.Write(); err != nil {
+				log.Infof("error updating okteto contexts: %v", err)
+				return nil, fmt.Errorf(errors.ErrCorruptedOktetoContexts, config.GetOktetoContextsStorePath())
+			}
+			return nil, fmt.Errorf(errors.ErrNotLogged, okteto.Context().Name)
+		}
+		if err != nil {
+			log.Info(err)
+			retries++
+			continue
+		}
+
+		// If userID is not on context config file we add it and save it.
+		// this prevents from relogin to actual users
+		if okteto.Context().UserID == "" && okteto.Context().IsOkteto {
```

Comment:
why wouldn't it be stored already?

Reply:
I just moved the code from use to create because is where this code belongs
There was a previous version where it was not saved and then we stored it for vscode plugin.